views/annotation/annotationoptions.html

```diff
@@ -35,13 +35,41 @@
           title="Show the Description field in the annotation">
         <input type="checkbox" class="custom-control-input" name="showAnnotationDescription"
             ng-attr-id="showAnnotationDescription{{ctrl.uid}}"
-            ng-change="ctrl.updateAnnotation()"
+            ng-change="ctrl.updatePreview()"
             ng-disabled="!ctrl.annotationOptions.show"
             ng-model="ctrl.annotationOptions.showDescription"/>
         <label class="custom-control-label" ng-attr-for="showAnnotationDescription{{ctrl.uid}}">
           Show Description
         </label>
       </div>
     </div>
+    <div class="form-row">
```

Comment:
As a space saving measure, could you move the new options to fill the extra space on the right side?

![space](https://user-images.githubusercontent.com/35077650/57546281-a7e9ba00-7319-11e9-987b-985f89bd63b9.png)